src/systems/filecoin_markets/deal/_index.md

```diff
@@ -23,8 +23,7 @@ Filecoin Storage Market Deal Flow
 
 # Receive Challenge
 
-- 5. Miners enter the `Challenged` status whenever `NotifyOfPoStChallenge` is called by the chain. Miners will then have X Epoch as the ProvingPeriod to submit a successful PoSt before `
-CheckPoStSubmissionHappened` is called by the chain. Miners can only get out the challenge with `SubmitPoSt` or `onMissedPoSt`.
+- 5. Miners enter the `Challenged` status whenever `NotifyOfPoStChallenge` is called by the chain. Miners will then have X Epoch as the ProvingPeriod to submit a successful PoSt before `CheckSurprisePoStSubmissionHappened` is called by the chain. Miners can only get out the challenge with `SubmitPoSt` or `onMissedSurprisePoSt`.
```

Comment:
`SubmitPoSt` is not an actor method. For clarity this should be "`SubmitSurprisePoSt` or `SubmitElectionPoSt`". Also delete the few other remaining references to `SubmitPost`.